src/components/MissionContainer.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMissions, allMissions } from '../redux/missions/missionsSlice';
+import MissionList from './MissionList';
+
+const MissionContainer = () => {
+  const missions = useSelector(allMissions);
+  const dispatch = useDispatch();
+
+  console.log(missions);
```

Comment:
[Optional] I appreciate the use of `console.log()` to help me see the state of the missions. You could consider adding it as a comment to prevent having the `console.log()` as part of our code history.